### ISSUE TITLE
Discord Crash

### ISSUE THREAD

**Issue by A6uh** [JRoy/discord-experiments-android#7]
The last 3 versions of the apk consistently crash when opening your own profile or settings. I've tried uninstalling and reinstalling, but it seems like the only solution is to revert to the older versions.

**Comment by A6uh** [comment]
Issue started on version: 192204
Last working version: 192203
Device: Pixel 7 Pro
Android: 14

**Comment by JRoy** [comment]
Are you using the new tabs UI?

**Comment by A6uh** [comment]
> Are you using the new tabs UI?

Yes on version 192203 (last working version), the tabs UI appears to be enabled by default.

**Comment by JRoy** [comment]
Okay will look into this, current workaround is to spam click your profile on application startup to be able to be get into settings and then toggle the `Tabs UI Experiment` to off.
